pkgs/development/python-modules/tern/default.nix

```
@@ -0,0 +1,55 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pyyaml
+, docker
+, dockerfile-parse
+, requests
+, stevedore
+, pbr
+, debut
+, regex
+, GitPython
+, prettytable
+, idna
+}:
+buildPythonPackage rec {
+  pname = "tern";
+  version = "2.5.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "606c62944991b2cbcccf3f5353be693305d6d7d318c3865b9ecca49dbeab2727";
+  };
+
+  preBuild = ''
+    cp requirements.{in,txt}
+  '';
+
+  # No tests
+  doCheck = false;
+
+  nativeBuildInputs = [
+    pbr
+  ];
+
+  propagatedBuildInputs = [
+    pyyaml
+    docker
+    dockerfile-parse
+    requests
+    stevedore
+    debut
+    regex
+    GitPython
+    prettytable
+    idna
+  ];
+
+  meta = with lib; {
+    description = "A software composition analysis tool and Python library that generates a Software Bill of Materials for container images and Dockerfiles.";
```

```suggestion
    description = "A software composition analysis tool and Python library that generates a Software Bill of Materials for container images and Dockerfiles";
```

see https://nixos.org/manual/nixpkgs/unstable/#var-meta-description